packages/react/src/dev.tsx

```diff
@@ -163,38 +163,90 @@ const ConfiguredDynamicFlow = () => {
 const BasicForm = () => {
   return (
     <ConfigurationProvider
+      logo="https://membersapp.checkatrade.com/static/media/logo.e930e0049182a5b0e5bf.png"
       factors={[
-        { method: "webauthn" },
-        { method: "email_link" },
-        { method: "otp_via_email" },
-        { method: "otp_via_sms" },
-        { method: "password" },
+        // { method: "webauthn" },
```

Comment:
Bunch of mess in this file, I'd rather not commit these changes

Reply:
None of these will be committed, I'll clean this up after I'm done testing